sunpy/map/sources/rhessi.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, data, header, **kwargs):
             self.meta['ctype2'] = 'HPLT-TAN'
 
         self.meta['waveunit'] = 'keV'
-        self.meta['wavelnth'] = [self.meta['energy_l'], self.meta['energy_h']]
+        self.meta['wavelnth'] = f"{self.meta['energy_l']}-{self.meta['energy_h']}"
```

Comment:
I have no idea if this is a good idea. It's definitely a breaking change, it's been in SunPy for 5 years, on the other hand it means you can now actually save a RHESSI map to FITS.

@ehsteve this was your fault, if you want to weigh in here? :stuck_out_tongue:

Reply:
Ok, I changed this fix, so that we only fix it on save to FITS. Everything is still terrible but at least we don't break this.